rlinf/data/io_struct.py

```diff
@@ -172,13 +178,14 @@ class RolloutResult:
     """
 
     num_sequence: int
+    group_size: int
```

Comment:
`group_size` is not easy to understand, can you add comments to explain a little bit?

Reply:
This is the same name as in the yaml config, which is the number of responses generated per prompt. If a comment is needed, we should add it in the yaml.